### ISSUE TITLE
Add Tooltip Feature to Editor Keys

### ISSUE THREAD

**Issue by azr-arch** [hrushikeshrv/mjxgui#26]
## Feature Request

I propose we add a tooltip feature to the keys in the MJXGUI editor. This would enhance the user experience by providing immediate information about each key.

## Details

When a user hovers over a key, a tooltip should appear displaying a brief description or name of the key. For example, hovering over the “Α” key would display a tooltip with the text “Alpha”.

This feature would be particularly useful for keys that represent complex mathematical symbols or functions, helping users understand their meaning without needing to refer to external resources.

**Comment by hrushikeshrv** [comment]
Hi @azr-arch, thanks for this issue. This makes sense. 👍🏻 

Right now we are changing the way the editor window's HTML is generated in issue #18. Once that issue is fixed, we could start work on this. Should I assign this to you?

**Comment by azr-arch** [comment]
@hrushikeshrv hey do you guys got a discord or smth ?

**Comment by hrushikeshrv** [comment]
Yeah, I've posted about this repository on the Hacktoberfest discord - https://discord.gg/hacktoberfest

**Comment by jatin-47** [comment]
Yeah, but I guess we should have a separate discord specifically for this project, as then we can openly discuss new features and add-ons with each other without discussing it directly here as an issue.

**Comment by azr-arch** [comment]
> Yeah, I've posted about this repository on the Hacktoberfest discord - https://discord.gg/hacktoberfest

i meant discord server for this repo ?

**Comment by hrushikeshrv** [comment]
I haven't created a discord server for this repo. Although it could be useful, I imagine that MJXGUI is not big enough yet that the server will be used long-term 😅 

It's likely that the server will just become inactive after a few days/weeks. However, if MJXGUI takes off and gets more and more traffic, I will definitely consider making a discord server!

**Comment by azr-arch** [comment]
@hrushikeshrv  is this ready for changes ?

**Comment by hrushikeshrv** [comment]
Yes @azr-arch, I think you can start working on this issue now. I have just added some contributing guidelines, there you will find the steps to follow if you are working on an issue that requires you to change the HTML. You can find those steps [here](https://github.com/hrushikeshrv/mjxgui/blob/main/CONTRIBUTING.md#working-with-the-editors-html).
